app/views/shared/sidebar/_projects.html.erb

```diff
@@ -1,35 +1,31 @@
 <ul>
-  <% if @projects_by_teams.present? then %>
-    <% @projects_by_teams.each do |team, projects| %>
-      <% projects.each do |project| %>
-
-        <% if (project_page? ||
-               sample_types_page_project? ||
-               sample_groups_page_project?) && project == @project %>
-          <li class="active" data-parent="candidate">
-            <span class="tree-link line-wrap no-indent">
-              <i class="no-arrow"></i>
-              <span data-project-id="<%= @project.id %>"
-                    title="<%= @project.name %>"><%= @project.name %></span>
-            </span>
-            <%= render partial: 'shared/sidebar/experiments',
-                      locals: { project: project } %>
-          </li>
-        <% else %>
-          <li data-parent="candidate">
-            <span class="tree-link line-wrap no-indent">
-              <i class="no-arrow"></i>
-                <%= link_to project.name,
-                            project_action_to_link_to(project),
-                            title: project.name,
-                            data: { project_id: project.id } %>
-            </span>
-            <%= render partial: 'shared/sidebar/experiments',
-                      locals: { project: project } %>
-          </li>
-        <% end %>
+  <% if @projects_tree.present? then %>
+    <% @projects_tree.each do |project| %>
+      <% if (project_page? ||
+             sample_types_page_project? ||
+             sample_groups_page_project?) && project == @project %>
```

Comment:
Probably we can get rid of these 2 checks: `sample_types_page_project?`, `sample_groups_page_project?`, right?

Reply:
Done